movement/io/poses_accessor.py

```diff
@@ -11,7 +12,7 @@
 xr.set_options(keep_attrs=True)
 
 
-@xr.register_dataset_accessor("poses")
+@xr.register_dataset_accessor("move")
 class PosesAccessor:
```

Comment:
Should we also rename the class itself to `MoveAccessor`? And the file accordingly to `move_accessor.py`?
I was also thinking of moving this out of the io folder into `movement/move_accessor.py` (since it will no longer have strictly io functionalities).

Do you think this can be done in this PR without ruining things too much (I'm aware that we are violating the "one PR - one change" rule here)?

Reply:
I think I should revert the poses -> move name change and leave these to its own PR. Created a [new issue](https://github.com/neuroinformatics-unit/movement/issues/118) for this.

Reply:
Good call, thanks!